Aspire Budgeting/PreviewProvider/MockProvider.swift

```diff
@@ -21,19 +21,25 @@ enum MockProvider {
                                availableTotal: AspireNumber(stringValue: "$0"),
                                budgetedTotal: AspireNumber(stringValue: "$1000"),
                                spentTotal: AspireNumber(stringValue: "$50"),
-                               progressFactor: 0.25))
+                               progressFactor: 0.25,
+                               categories: [
+                                Category(row: ["", "", "Dummy Category", "$30", "b", "c", "20", "e", "f", "15"]), ]))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/16f63f2343d9e7b69eb70c1fc9896bed3fcc7422/Aspire Budgeting/PreviewProvider/MockProvider.swift#L26"></span>Line should be 100 characters or less: currently 117 characters<br />
<code>line_length</code> <code>MockProvider.swift:26</code></td>
    </tr>
  </tbody>
</table>